.github/workflows/build.yaml

```diff
@@ -0,0 +1,93 @@
+name: Build
```

Comment:
Since this is CI and not a "build" per-say, should we use `ci.yml` and `name: CI`?

Reply:
Works for me. Updated in latest.